CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.2.0
+  - Encode tags as fluent forward protocol tags
```

Comment:
minor: If there is a corresponding issue, it is nice to like to it with #num convention

Reply:
Aye - yes, good catch.